core/src/main/scala/akka/persistence/r2dbc/internal/R2dbcExecutor.scala

```diff
@@ -127,6 +127,43 @@ class R2dbcExecutor(val connectionFactory: ConnectionFactory, log: Logger)(impli
   private def getConnection(): Future[Connection] =
     connectionFactory.create().asFuture()
 
+  /**
+   * Run DDL statement with auto commit.
+   */
+  def executeDdl(logPrefix: String)(statement: Connection => Statement): Future[Done] = {
+    val startTime = System.nanoTime()
+
+    getConnection().flatMap { connection =>
+      connection.setAutoCommit(true).asFutureDone().flatMap { _ =>
+        val done =
+          try {
+            val stmt = statement(connection)
+            stmt.execute().asFuture().flatMap { result =>
+              result.getRowsUpdated.asFutureDone()
```

Comment:
somewhat surprising but it seems like ddl statements doesn't return the number of rows so when I tried to use the ordinary `updateOne` it failed with:
```
Publisher [MonoHandleFuseable] completed without first value.
```

Reply:
I think we got tripped elsewhere previously on the fact that Monos are more like `Future[Option[T]]` than `Future[T]`.